src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: rectangle ,");
+        super.draw();
+    }
```

Comment:
It seems like the `Rectangle` class is inheriting from `Quadrangle` and overriding the `area` and `draw` methods without changing their behavior. If the implementations of `area` and `draw` methods in the `Quadrangle` class are already suitable for a rectangle, there is no need to override them here. This could violate the checklist item: 'Don't put all behavior into a single interface if the methods are conceptually different from each other.' Make sure that each class is only responsible for its specific behavior and that you are not duplicating code unnecessarily.